common/src/main/resources/assets/minecraft_access/lang/en_us.json

```
@@ -298,5 +298,7 @@
   "minecraft_access.wolf_variant.striped": "Striped",
   "minecraft_access.wolf_variant.woods": "Woods",
   "minecraft_access.xp_indicator.decreased": "Player XP level decreased to %s",
-  "minecraft_access.xp_indicator.increased": "Player XP level increased to %s"
+  "minecraft_access.xp_indicator.increased": "Player XP level increased to %s",
+  "minecraft_access.hudStatus.announce_shown": "Hud shown",
+  "minecraft_access.hudStatus.announce_hidden": "Hud hidden"
```

Nit: `hudStatus` should be in snake_case to be consistent with the rest of the keys and HUD should be capitalised.
```suggestion
  "minecraft_access.hud_status.announce_shown": "HUD shown",
  "minecraft_access.hud_status.announce_hidden": "HUD hidden"
```